numba/cpython/numbers.py

```diff
@@ -361,6 +361,38 @@ def int_ne_impl(context, builder, sig, args):
     return impl_ret_untracked(context, builder, sig.return_type, res)
 
 
+def int_su_cmp(op):
```

Comment:
Perhaps:

```suggestion
def int_signed_unsigned_cmp(op):
```

would make it a bit easier to guess what the function does from the name? (I had to read the description in the implementation to figure out that s = signed and u = unsigned)

Reply:
No problem. I had it this way to match the other comparison functions.